packages/lab/src/form-field-next/FormFieldLabel.tsx

```diff
@@ -0,0 +1,33 @@
+import { clsx } from "clsx";
+import { Label, makePrefixer, TextProps } from "@salt-ds/core";
+
+import "./FormFieldLabel.css";
+
+const withBaseName = makePrefixer("saltFormFieldLabel");
+export interface FormFieldLabelProps
+  extends Omit<TextProps<"label">, "variant" | "styleAs"> {
+  /**
+   * Whether the Form Field is disabled.
+   */
+  disabled?: boolean;
+  /**
+   * The label value for the FormFieldLabel
+   */
+  label?: string;
+}
+
+export const FormFieldLabel = ({
+  className,
+  disabled,
+  label,
+  ...restProps
+}: FormFieldLabelProps) => (
+  <Label
```

Comment:
This label is not currently hooked up the control inside the FormField

Reply:
I think this is fixed, but may be missing something ?

Reply:
It's fixed now